server/konnectors/edf.coffee

```diff
@@ -0,0 +1,1347 @@
+xml2js = require 'xml2js'
+request = require 'request'
+async = require 'async'
+moment = require 'moment'
+cozydb = require 'cozydb'
+File = require '../models/file'
+
+parser = new xml2js.Parser()
+builder = new xml2js.Builder headless: true
+
+# TODO :
+# Move updateOrCreate in a lib
+# multi account
+# multi contract
+
+# Models
+Client = cozydb.getModel 'Client',
+    clientId: String
+    vendor: String
+    numeroAcc: String
+    address: Object
+    name: Object
+    email: String
+    telMobile: String
```

Comment:
Yes it's ugly sometimes. I will have check if I can fix some of them (as the ones you just mention). But some other are so specific, specific to crafts I don't mastery, or related to properties which don't exist out of France, that I fear that the translation may make them meaningless.